ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -82,6 +84,9 @@ define([
             // check for youtube video atoms
             youtube.checkForVideos();
 
+            // if there is a contributions epic, track it
+            creativeInjector.trackLiveBlogEpic();
+
```

Comment:
I should've added earlier that this will also need to go into 'liveblogLoadMore' (called when a user taps the more button to load in old blocks), sorry I didn't think of that before! 

It may be sensible to refactor out twitter.checkForTweets(), youtube.checkForVideos(), creativeInjector.trackLiveBlogEpic() into a function and call that from liveblogLoadMore and liveblogNewBlockDump so we don't have to do these updates in two places.